dotcom-rendering/src/web/components/Snap.tsx

```diff
@@ -10,23 +10,19 @@ type Props = {
 };
 
 export const Snap = ({ enriched }: Props) => {
-	return (
-		<div css={snapStyles}>
-			{enriched?.embedCss !== undefined && (
-				<Global styles={enriched?.embedCss} />
-			)}
-
-			{enriched?.embedHtml !== undefined && (
-				<div
-					dangerouslySetInnerHTML={{ __html: enriched?.embedHtml }}
-				/>
-			)}
+	if (enriched?.embedHtml === undefined) {
+		return <></>;
+	}
 
-			{enriched?.embedJs !== undefined && (
-				<script
-					dangerouslySetInnerHTML={{ __html: enriched?.embedJs }}
-				/>
-			)}
-		</div>
+	return (
+		<div
+			css={[
+				snapStyles,
+				css`
```

Comment:
I'm not sure if you're meant to be able to do this with emotion but it seems to work for sandboxing CSS!

Reply:
Well, what do you know, that's actually kinda neat